docs/WritingTests.asciidoc

```diff
@@ -158,17 +338,23 @@ sub run() {
 }
 --------------------------------------------------------------
 
-In addition to Perl, it is possible to write tests in Python. Although the
-boilerplate is slightly different the function names are the same.
+=== Example Python test modules
+[id="testmodule_python_examples"]
+
+The following examples are short complete test modules written in Python
+implementing the interface described above.
+
+==== openQA web UI sample test
+[id="testmodule_python_webui"]
 
 [caption="Example: "]
 .Test for the openQA web UI written in Python
+
 [source,python]
 --------------------------------------------------------------
 from testapi import *
 
-
-def run(self):
+def run():
```

Comment:
```suggestion
def run(self):
```
Otherwise we'll get
```
TypeError: run() takes 0 positional arguments but 1 was given
```

Reply:
Changing it! 

just out of my own curiosity, can you:

```
def run(self):
  print(type(self), dir(self), self)
```

I wanna know how the self looks like :thinking: